### PR TITLE
Added adoption script docs

### PR DESCRIPTION
**Pull request by kgal-pan**
## Status
On Hold (waiting for approval of https://github.com/demisto/content/pull/19115)

## Related Issues
https://github.com/demisto/content/pull/19115

## Description
Added another adoption option using the command line.

### CODE REVIEW

File: docs/partners/adopt.md [hunk 1]
```diff
@@ -143,3 +139,15 @@ Once the Cortex XSOAR engineering team merges your pull request, you will have s
...
+<br/>
+<strong>Command Line Instructions</strong>
+<details>
+<summary>Click here to see the command line instructions</summary>
+
+From within the `content` directory, run:
+```bash
+./Utils/adopt_pack.bash
+```
+
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:153]:
```suggestion
This script will automatically perform the necessary steps to create an adoption PR.

Requirements:
Before using this automation, make sure you have [Git](https://git-scm.com/downloads) and [Python](https://www.python.org/downloads/) installed.

Note:
This script will install `demisto-sdk` python package if it does not exist in your environment.

Usage:
1. After [Forking](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-2-fork-the-github-repo) and [Cloning](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-3-clone-the-github-fork-locally) the Content repo to your local environment, you will need to search for your pack name. Go to the `Packs` folder inside the cloned repo, and search for your pack folder, the name of the directory is the pack name that we will use to run the automation with.
2. From within the `content` directory, run:
\```bash
./Utils/adopt_pack.bash "start" "MyPackName"
\```
3. Visit the link given in the output of the script to publish the pull request containing the adoption details created by the automation.  

```

File: docs/partners/adopt.md [hunk 2]
```diff
@@ -28,19 +28,17 @@ The process to Adopt-a-Pack takes under 5 minutes! Follow the step by step instr
...
 
     ```
-    Note: Support for this pack moved to the partner on MONTH, DAY, YEAR.
-    
-    Please contact the partner directly via the support link on the right.
+    Note: Support for this Pack will be moved to the Partner on MONTH, DAY, YEAR.
     ```
     Make sure you change the `MONTH`, `DAY`, and `YEAR` to the appropriate date that is 90 days from your submittal date.
 
     - Once complete, save these changes and run `demisto-sdk update-release-notes -i <path to pack> -f` to update the release notes. See [documentation](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md). After the command has been completed, it will create the new release note Markdown file in the `ReleaseNotes` folder and update the version number in `pack_metadata.json`. Before continuing, you need to add the following text to the release note: `Start of adoption process, update to readme file`
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:41]:
```suggestion
    - Once complete, save these changes and run `demisto-sdk update-release-notes -i <path to pack> -f` to update the release notes. See [documentation](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md). After the command has been completed, it will create the new release note Markdown file in the `ReleaseNotes` folder and update the version number in `pack_metadata.json`. Before continuing, you need to add the following text to the release note: `Start of adoption process, visit the pack readme for more info`
```

Comment (ShahafBenYakir) [reply docs/partners/adopt.md:41]: Let's also fix this ☝️  in the Github UI flow

**Comment by ShahafBenYakir** [comment]
BTW we have we changed the first category to "vs-code" it should be "local dev environment" or something of that sort.

File: docs/documentation/release-notes.md [hunk 3]
```diff
@@ -207,7 +207,7 @@ In some cases, a new version is introduced which breaks backward compatibility.
 From Cortex XSOAR version 6.8 and above, there is support to mark a new version as a **breaking changes version**.
 
 Marking a version as a **breaking changes version** provides the user with an alert before installation:
-![image](../doc_imgs/integrations/bc_alert_example.png)
+![image](/doc_imgs/integrations/bc_alert_example.png)
```
Comment (ShahafBenYakir) [inline docs/documentation/release-notes.md:210]: Why is this change necessary?

File: docs/partners/adopt.md [hunk 4]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
 
-**After the 90 days, another pull request must be submitted to complete the adoption process. Please follow the below steps if you are submitting the final pull request via Visual Studio:**
+```
+Start of adoption process.
+```
+
+3. Now, it's time to save and commit the changes as a GitHub pull request. Once you publish the changes via Visual Studio, GitHub will prompt you to open a pull request. When prompted, click the green button "Compare & pull request".
+    - Double check the pull request to ensure all changes are correct.
+    - Change the pull request title to 'Company Name Pack Adoption' and adjust the description to _Updating README file for adoption_.
+    - When ready, click the green button “Create pull request" on the bottom of the page.
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:96]:
```suggestion
    - When ready, click the green button `Create pull request` on the bottom of the page.
```

File: docs/documentation/release-notes.md (hunk 3 above)
Comment (kgal-pan) [reply docs/documentation/release-notes.md:210]: I think this occurred in one of the merges from `master`, like [this one](https://github.com/demisto/content-docs/pull/1103/commits/1021fbb5f0cdfb820f4a8628a48b5c7184c89f74). From what I saw, the build actually automatically corrects the paths.

I see that the image loads fine in https://xsoar-pan-dev--pull-request-1103-mma9v2uu.web.app/docs/documentation/release-notes. Should I revert or leave as is?

File: docs/partners/adopt.md [hunk 5]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
+    Make sure you change the `MONTH`, `DAY`, and `YEAR` to the appropriate date that is 90 days from your submission date.
+
+4. Once complete, save these changes and run `demisto-sdk update-release-notes -i <path to pack> -f` to update the release notes. See [documentation](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md). After the command has been completed, it will create the new release note Markdown file in the `ReleaseNotes` folder and update the version number in `pack_metadata.json`. Before continuing, you need to add the following text to the release note: 
+
+        
+        Start of adoption process.
+        
+
+5. Edit the Pull Request title to '`COMPANY_NAME` Pack Adoption' and adjust the description to 'Updating README file for adoption'.
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:148]:
```suggestion
5. At the bottom of the screen, edit the Pull Request title to '`COMPANY_NAME` Pack Adoption' and adjust the description to 'Updating README file for adoption'.
```

File: docs/partners/adopt.md [hunk 6]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
+6. Create a new branch named `partners-COMPANY_NAME-adoption-start`.
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:149]:
```suggestion
6. Create a new branch named `partner-COMPANY_NAME-adoption-start`. 
```

File: docs/partners/adopt.md [hunk 7]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
 
-Please follow the below steps if you are working on a cloned Github repository from an IDE: 
+<br/>
+<details><summary><strong>Adopt Using Helper Script</strong></summary>
+
+<br/>
+
+<div>This script will automatically perform the necessary steps to create an adoption PR.
+
+**Note:** The script was only tested on Unix systems, specifically Mac OS and Ubuntu. It might not work correctly on other systems.
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:39]: I don't relate to this disclaimer as it doesn't give anything to the user besides the feeling we are not taking responsibility if this fails. Instead, I think we should mention that the script is supported in Linux and Mac OS and if there's an issue the user should consult his Rep. from the PS team.

File: docs/partners/adopt.md [hunk 8]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
+<div>This script will automatically perform the necessary steps to create an adoption PR.
+
+**Note:** The script was only tested on Unix systems, specifically Mac OS and Ubuntu. It might not work correctly on other systems. 
+
+**Requirements:**
+Before using this automation, make sure you have [`git`](https://git-scm.com/downloads) and [`python3`](https://www.python.org/downloads/) installed and in your `PATH`. This script will install [`demisto-sdk`](https://github.com/demisto/demisto-sdk#installation) python package if it does not exist in your environment.
+
+Follow the steps below to adopt using the helper script:
+
+1. Change directory to where the `content` repository is located. This is the location where you [cloned the forked `content` repository](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-3-clone-the-github-fork-locally).
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:46]:
```suggestion
1. Inside your terminal, change your working directory to the root of the Content repository. This is the location where you [cloned the forked `content` repository](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-3-clone-the-github-fork-locally).
```

File: docs/partners/adopt.md [hunk 9]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
+
+2. Look for the Pack you want to adopt in the `content/Packs/` directory. You will use the folder name as the second argument to the `adopt_pack.bash` script.
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:48]: They should already be in content folder
```suggestion
2. Look for the pack you want to adopt under the `Packs/` directory. You will use the folder name as the second argument to the `adopt_pack.bash` script. 
```

**Review by ShahafBenYakir — COMMENTED** [review]
The Script part looks good.
I started reviewing the GitHub Ui flow and TBH I think it is too much, I think it will be easier if we use the GitHub VS code extension for this as you can perform all the changes in one commit instead of going back and forth between the PR and each file. WDYT?

File: docs/partners/adopt.md [hunk 10]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
+
+</div>
+</details>
+
+<br/>
+
+<details><summary><strong>Adopt Using Visual Studio Code</strong></summary>
+<br/>
+<div>
+If you are creating the Pull Request from Visual Studio Code, please follow the below steps. For additional guidance, watch the <a href="https://www.youtube.com/watch?v=9GPkhtRw4Oc">Visual Studio video overview</a>.
```
[the remark this answers is not in the document]
Comment (kgal-pan) [reply docs/partners/adopt.md:87]: I thought what we talked about was having a parent section with two subsections, first one should be the script, second VSCode. 

I think the VSCode video isn't needed unless someone decides they want to do the changes locally and but using the IDE.

File: docs/partners/adopt.md [hunk 11]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
-
-
 2. Next, go to the pack_metadata.json file and update the following sections:
-    - currentVersion - update the version. For this example, we would be updating it to 1.2.12
-    - support - must say `partner`
-    - author - must say your company name
-    - url - must be changed to your company’s support site
-    - email - must be your company's support email 
-    
+    - `currentVersion` - update the version. For this example, we would be updating it to 1.2.12
```
[the remark this answers is not in the document]
Comment (kgal-pan) [reply docs/partners/adopt.md:116]: It's referring to the video tutorial where they update the Pack version in `pack_metadata.json`.

File: docs/partners/adopt.md [hunk 12]
```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
...
-  - **NOTE:** If you are not ready to officially submit the pull request for review, you can create a draft pull request instead. To the right of the              “Create pull request” button there is a small button with an arrow, click that and choose the Draft option. This will still create the pull request        but the XSOAR eng team will not review it until it is taken out of draft.
+    Make sure you change the `MONTH`, `DAY`, and `YEAR` to the appropriate date that is 90 days from your submission date.
+
+4. Once complete, save these changes and run `demisto-sdk update-release-notes -i <path to pack> -f` to update the release notes. See [documentation](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md). After the command has been completed, it will create the new release note Markdown file in the `ReleaseNotes` folder and update the version number in `pack_metadata.json`. Before continuing, you need to add the following text to the release note: 
+
+        
+        Start of adoption process.
+        
+
```
[the remark this answers is not in the document]
Comment (kgal-pan) [reply docs/partners/adopt.md:156]: Nice catch! 👀

File: docs/partners/adopt.md [hunk 13]
```diff
@@ -20,126 +20,257 @@ Adopting a pack has several advantages for the partner:
 ![pack example cyren](/doc_imgs/partners/packexample_cyren.png)
 
 ## Process
-The process to Adopt-a-Pack takes under 5 minutes! Follow the step by step instructions and watch the video tutorials below to learn how to submit your adoption pull requests.  
+Choose one of the options below and follow the step by step instructions or watch the video tutorials below to learn how to submit your adoption Pull Requests. 
 
-<strong>Visual Studio Instructions</strong>: If you are creating the pull request from Visual Studio, please follow the below steps. For additional guidance, watch the <a href="https://www.youtube.com/watch?v=9GPkhtRw4Oc">Visual Studio video overview</a>.
-<details>
-<summary>Click here to see the Visual Studio instructions</summary>
+<details><summary><strong>Working on a local clone of Content repo</strong></summary>
```
Comment (ShahafBenYakir) [inline docs/partners/adopt.md:25]: The nested summary does not look so good as it is not clear what belongs to what and ther's no indentation between sections (see attached gif). 
ezgif com-gif-maker (1)

Let's Instead change this to a simple `###` header, like this:
```suggestion
### Option 1: Work on a local clone of the Content repo
These methods assume that you have already [forked the Content repository](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-2-fork-the-github-repo) and [cloned the fork onto your local machine](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-3-clone-the-github-fork-locally).
Choose either one of the methods below to start or complete adoption.

```
And also add an additional header before the [Adopt Using GitHub UI](https://github.com/demisto/content-docs/pull/1103/files#diff-d12af46ecf104a4963eef8a8a31a022f0e69642d0816056c90032653ce98a813R202), such that finally we'll have two headers under the first there will be the `Adopt Using Helper Script` and `Adopt Using a Text Editor/IDE` and under the second there will be the `Adopt Using GitHub UI`

Comment (kgal-pan) [reply docs/partners/adopt.md:25]: Yea, I tried to do nested section but seems that docusaurus fails when using `blockquotes` tag to create a tabbed sub-section. I then used [their documentation ](https://docusaurus.io/docs/markdown-features#details) but as you can see, the nesting doesn't actually render as expected.

Will use h3 sections instead as suggested.

**Review by ShahafBenYakir — COMMENTED** [review]
@kgal-pan why have we detached the `<summary>` collapsable feature? The idea was to have two H3 headers `Working on a local clone of Content repository` and `Adopt Using GitHub UI`. Under the first, there should be two collapsable sections `Adopt Using Helper Script` and `Adopt Using a Text Editor/IDE`, and under the second H3 header there should be a collapsable section with the Github UI details.

**Review by ShahafBenYakir — APPROVED** [review]
Awsome!
